httpcache.go

```diff
@@ -249,6 +256,19 @@ func parseCaddyfileGlobalOption(h *caddyfile.Dispenser, _ interface{}) (interfac
 					switch directive {
 					case "basepath":
 						apiConfiguration.BasePath = h.RemainingArgs()[0]
+					case "prometheus":
+						apiConfiguration.Prometheus = configurationtypes.APIEndpoint{}
+						for nesting := h.Nesting(); h.NextBlock(nesting); {
+							directive := h.Val()
+							switch directive {
+							case "basepath":
+								apiConfiguration.Prometheus.BasePath = h.RemainingArgs()[0]
+							case "enable":
+								apiConfiguration.Prometheus.Enable, _ = strconv.ParseBool(h.RemainingArgs()[0])
+							case "security":
+								apiConfiguration.Prometheus.Security, _ = strconv.ParseBool(h.RemainingArgs()[0])
+							}
+						}
```

Comment:
Basically this is what I'm suggesting, as per the previous comment

```suggestion
					case "prometheus":
						apiConfiguration.Prometheus = configurationtypes.APIEndpoint{}
						apiConfiguration.Prometheus.Enable = true
						for nesting := h.Nesting(); h.NextBlock(nesting); {
							directive := h.Val()
							switch directive {
							case "basepath":
								apiConfiguration.Prometheus.BasePath = h.RemainingArgs()[0]
							case "security":
								apiConfiguration.Prometheus.Security, _ = strconv.ParseBool(h.RemainingArgs()[0])
							}
						}
```